src/harness/testcases/WINNF_FT_S_FAD_testcase.py

```diff
@@ -229,6 +229,10 @@ def generate_FAD_1_default_config(self, filename):
                                                               pal_low_frequency,
                                                               pal_high_frequency,
                                                               device_a['userId'])
+
+      device_a['installationParam']['latitude'], device_a['installationParam'][
+        'longitude'] = getRandomLatLongInPolygon(ppa)
+      ppa['cbsdClusterList'] = [{'cbsdSerialNumber': device_a['cbsdSerialNumber'] , 'fccId': device_a['fccId']}]
```

Comment:
Please implement consistent with GRA.4.

Reply:
thank you for you comments, I updated the code.